### ISSUE TITLE
Delete impossible on PostGIS with lizmap 3.1.11

### ISSUE THREAD

**Issue by mathieubossaert** [3liz/lizmap-web-client#859]
Hi,
I try to delete a row in a child/related table and I get the error "Lizmap cannot get the feature data with WFS"

![capture d ecran_2018-05-24_09-43-21](https://user-images.githubusercontent.com/1642645/40471482-1df9a6d0-5f37-11e8-9898-b867bbc7ad68.png)
When I insert or try to delete a rox, lines appear in lizmap/var/log/error.log : 
`2018-05-24 09:28:06	37.173.55.40	notice	2018-05-24 09:28:06	[8]	json_decode(): integer overflow detected	/var/www/websig/lizmap/modules/lizmap/controllers/edition.classic.php	258`
Thanks a lot.

**Comment by rldhont** [comment]
Can you provide a part of the GeoJSON ?

**Comment by rldhont** [comment]
@mathieubossaert this is it https://stackoverflow.com/questions/43815484/warning-json-decode-integer-overflow-detected ?

**Comment by mathieubossaert** [comment]
Maybe it could be a bbox problem : 
```json

{"type": "FeatureCollection",
 "bbox": [ -179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368, -179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368, 179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368, 179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368],
 "features": [
  {
   "type":"Feature",
   "id":"suivi_parcelle.10402@@2018",
   "geometry":null,
   "properties":{
      "id":10402,
      "annee_culturale":2018,
      "conduite":"biologique",
      "culture":"Vigne",
      "preconisations":null,
      "preconisations_iae":null,
      "gid":2439
   }
}
 ]
}
```

**Comment by mathieubossaert** [comment]
OK I changed QGIS-server and back to 2.14 it works.

**Comment by rldhont** [comment]
I have fixed QGIS Server 2.18 and 3.2 for this integer overflow!
* 3.2 https://github.com/qgis/QGIS/commit/c8a5992b2b85b061823a0b32013bc7289d9015f7
* 2.18 https://github.com/qgis/QGIS/commit/efce338b52485c6c7caf9cb8a971caf83b5cd5fa

**Comment by rldhont** [comment]
Will be available in 2.18.21 on friday 22th of june.
